src/main.ts

```diff
@@ -382,6 +383,12 @@ export async function createPipeline(config: Config): Promise<string> {
   if (typeof process.env.VIB_PUBLIC_URL === "undefined") {
     core.setFailed("VIB_PUBLIC_URL environment variable not found.")
   }
+  const verificationModeValues = ["PARALLEL", "SERIAL"]
+  core.info("verificationModeValues")
+  if (!verificationModeValues.includes(config.verificationMode)) {
+    core.setFailed("The value of Verification Mode is not valid, the default value will be used.")
```

Comment:
```suggestion
    core.setFailed(`The value ${config.verificationMode} for verification-mode is not valid, the default value will be used.`)
```

Reply:
Needs to be a warning. If you do setFailed, that immediately stops the GitHub Action

Reply:
Done